apollo-router/src/plugins/telemetry/mod.rs

```diff
@@ -472,7 +445,10 @@ impl Telemetry {
                 } else if atty::is(atty::Stream::Stdout) {
                     let telemetry = tracing_opentelemetry::layer().with_tracer(tracer);
 
-                    let subscriber = sub_builder.finish().with(telemetry);
+                    let subscriber = sub_builder
+                        .event_format(formatter::TextFormatter::new())
```

Comment:
does this change they way events are reported when using trace aggregators like jaeger?

Reply:
I wouldn't expect it to, but I'm not sure how to verify it. I'll make sure to verify this is unaffected before I merge.